src/core/services/theming/theming.js

```diff
@@ -69,6 +69,20 @@ function detectDisabledThemes($mdThemingProvider) {
  *   });
  * </hljs>
  *
+ * ### Dyanmic Themes Target
+ *
+ * By default, the dynamic styles are always appended to the head section.<br />
+ * If you want to change the target, so the styles are appended to a particular element instead, you could use this option.<br />
```

Comment:
Can you please wrap this line at 100 chars as documented in the [coding guidelines](https://github.com/angular/material/blob/master/docs/guides/CODING.md)?